src/components/Header.js

```diff
@@ -21,7 +21,7 @@ function Header ({
 			{customControls ? customControls : <span />}
 			{!!showCloseButton && (
 				<button
-					title={closeTitle || 'Close (Esc)'}
+					title={closeButtonTitle || 'Close (Esc)'}
```

Comment:
Actually you can just pass that straight through, no need to default in the <Header /> component because it's rendered by <Lightbox /> and should just inherit closeButtonTitle